packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "10.6.2",
+  "version": "11.0.0",
```

Comment:
No one is using this component yet, so I waffled on whether to make this a breaking change, but a component name change should be a breaking change whether anyone is using it or not.